common/changes/@autorest/core/change-trenton-to-terraform_2021-03-02-07-03.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@autorest/core",
+      "comment": "change trenton to terraform",
+      "type": "minor"
```

would maybe just do a `patch` here instead of bumping the minor specially as this is not a public package yet.
```suggestion
      "type": "patch"
```